semgrep/semgrep/target_manager.py

```diff
@@ -113,36 +128,26 @@ def _expand_dir(
                         encoding="utf-8",
                         stderr=subprocess.DEVNULL,
                     )
-                except subprocess.CalledProcessError:
-                    raise NotGitProjectError(
-                        f"{curr_dir.resolve()} is not a git repository."
-                    )
 
-                tracked = TargetManager._parse_output(tracked_output, curr_dir)
-                untracked_unignored = TargetManager._parse_output(
-                    untracked_output, curr_dir
-                )
+                    tracked = _parse_output(tracked_output, curr_dir)
```

Comment:
We should put these in the `try`'s `else` block to minimize the exception catching surface. That way we don't catch exceptions we didn't mean to, and it's clear where the exception is expected to come from. Not that I would expect that to happen here, but I think it's good practice to minimize the amount of code in a `try` block.